src/main/java/net/glowstone/util/TickUtil.java

```diff
@@ -0,0 +1,46 @@
+package net.glowstone.util;
+
+/**
+ * Utility class to ease conversion between minutes/seconds and ticks.
```

Comment:
Since this class only does SI -> ticks, change this to "from standard time units to in-game ticks."